src/app/contextBroker/contextBroker.cpp

```diff
@@ -783,495 +783,393 @@ static const char* validLogLevels[] =
 
```

Comment:
No real reason to do all this work in a separate branch, pretty much only me working on the broker anyway ... :-)
It is not a problem to me to finish every PR leaving the broker in a 100% working state.
There is nothing about "half-working functionalities" here, only refactors.
Doing it in a feature branch would only add extra work.

The next step is to remove the 'payloadWord' from the RestService vector and after that more simplifications of the vector

Reply:
In that case, then CHANGES_NEXT_RELEASE entry should be "inaugurated" in the first PR addressing this, eg.:

```
- Hardening: refactor request routing logic (#3109, step 1)
```

or a similar wording.

Reply:
There is no impact on the usage of the broker ...
Normally, when we refactor stuff we don't mention it in the CNR.
The only thing users will notice is that the broker is faster after these changes ...

Reply:
I agree there is no impact for the user from a functional perspetive, but even in that case it would be interesing to mark it in the changelog (specially when there could be a potentiall impact, non-functional, in the form of performance increase). That's way we use `Hardening` as prefix instead of `Add` or `Fix`. It doesn't hurt and may be useful.

Note we have already done this in the past, eg:

```
- Hardening: NGSIv1 responses don't use pretty-print JSON format any longer, as NGSIv2 responses work (potentially saving around 50% size) (#2760)
```

Reply:
Fixed in 74c7a9c8d